test/box/net.box_reconnect_after_gh-3164.test.lua

```diff
@@ -51,3 +58,7 @@ collectgarbage('collect')
 -- Now weak.c is null, because it was weak reference, and the
 -- connection is deleted by 'collect'.
 weak.c
+
+-- Since several tests may run on the same Tarantool instance,
+-- enabling JIT back.
+jit.on()
```

Comment:
1. AFAIK it is not so since https://github.com/tarantool/test-run/commit/33fbace8307e64e66b0b2178ac952b2697673917.
2. Is it possible that a platform specific default is JIT disabled?

Reply:
I dropped this part then. Thanks!